packages/array-mean/src/index.js

```
@@ -0,0 +1,22 @@
+/**
+ * Computes the mean of the given values
+ * @param {Array<number>} input
+ * @return {number}
+ */
+export default function mean(input) {
+    if (!Array.isArray(input)) {
+        throw new Error('input must be an array');
+    }
+
+    var sum = 0;
+    var l = input.length;
+
+    if (l === 0) {
+        return undefined;
```

I think that's better to throw an error